scripts/GetNetstandardAssembly.ps1

```diff
@@ -0,0 +1,50 @@
+##############################
+#.SYNOPSIS
+# Gets the latest dotnet (core) SDK's netstandard.dll path.
+#
+#.DESCRIPTION
+# Locates the latest dotnet SDK netstandrd.dll such as $SdkInstallDir\<version>\Microsoft\Microsoft.NET.Build.Extensions\net461\lib\netstandard.dll.
+#
+#.EXAMPLE
+# PS:> .\GetNetstanardAssembly.ps1
+# C:\Program Files\dotnet\sdk\2.1.202\Microsoft\Microsoft.NET.Build.Extensions\net461\lib\netstandard.dll 
+# 
+#
+#.NOTES
+# This script is designed from - https://github.com/PowerShell/PowerShell/blob/master/docs/cmdlet-example/command-line-simple-example.md#the-fix-for-missing-netstandarddll
+##############################
+[CmdletBinding()]
+[OutputType([string])]
+param
+(
+	# Path to SDK install directory
+    [ValidateNotNullOrEmpty()]
+    [string] $SdkInstallDir = 'C:\Program Files\dotnet\sdk\',
+	
+	# Version to check major and minor to match and build has to be less than
+	[string] $MajorMinorSDKVersionCheck = '2.1.300'
+)
+
+# .NET CORE 2.0 Downloads - https://www.microsoft.com/net/download/dotnet-core/2.0
+# .NET CORE 2.1 Downloads - https://www.microsoft.com/net/download/dotnet-core/2.1
+
+$versionCheck = [version]$MajorMinorSDKVersionCheck
+$sdkDir = Get-ChildItem $SdkInstallDir -Directory | Where-Object Name -Match '\d\.\d\.\d' | ForEach-Object { 
+    $version = [version]$_.BaseName
+    $_ | Add-Member -Name SDKVersion -MemberType NoteProperty -Value $version
+    $_ 
+} | Where-Object { $_.SDKVersion.Major -eq $versionCheck.Major -and $_.SDKVersion.Minor -eq $versionCheck.Minor -and $_.SDKVersion.Build -lt $versionCheck.Build } | Sort-Object SDKVersion -Descending | Select-Object -First 1
+
+if(!$sdkDir) {
+	throw "Unable to find SDK directory under: $SdkInstallDir"
```

Comment:
Consider adding the version check number here so readers of the error message know that it failed to find a compatible version. Currently this error can be misread as not having the SDK installed at all.

Reply:
Yeah I will add